scss/custom/_variables.scss

```diff
@@ -43,3 +43,11 @@ $tooltip-opacity: 0.9;
 // Fonts! (TODO)
 $font-family-base: 'Source Sans Pro', sans-serif;
 $font-family-alt: 'Work Sans', sans-serif;
+
+// Can be removed once Bootstrap is dropped
```

Comment:
This comment is a little confusing. We're using some of these variables outside out bootstrap (e.g. `$black` is used all over `main.scss`) so I don't want anyone to accidentally delete these in the future.

Can you remove the comment?

Reply:
Yeah, that makes sense.